src/CommunityToolkit.Maui.Core/Views/DrawingView/IDrawingView.shared.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.ObjectModel;
+using System.Windows.Input;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The DrawingView allows you to draw one or multiple lines on a canvas
+/// </summary>
+public interface IDrawingView : IView
+{
+	/// <summary>
+	/// Event occurred when drawing line completed
+	/// </summary>
+	/// <param name="drawingLineCompletedEventArgs">Last drawing <see cref="ILine"/></param>
+	void DrawingLineCompleted(DrawingLineCompletedEventArgs drawingLineCompletedEventArgs);
```

Comment:
Using an `EventArgs` type as a parameter we influence implementors to use `EventHandlers`, I believe that we should think of another approach or just use an `IEnumerable<ILine>`, what do you think?

Reply:
I have a problem with it. Originally it was a ILine, but I can't convert from MauiDrawingLine to ILine, that is why I pass all parameters here.

Reply:
I can create a DTO for it instead of EventArgs